### ISSUE TITLE
Feature request: Accessibility Audit - Adding images of failing elements in report.

### ISSUE THREAD

**Issue by joelhsmith** [GoogleChrome/lighthouse#3984]
Google Chrome natively supports taking screenshots of individual elements. https://umaar.com/dev-tips/156-element-screenshot/

Since the Axe engine supplies the path to any failing element, could we use Chrome's native screenshot capability to retrieve screenshots of those elements added in the context of the report?

Example: Button element that has failing contrast ratio gets a screenshot of that element.

I know that this could take an image of the entire page, but maybe it could be converted into a thumbnail image?

**Comment by patrickhulce** [comment]
While we agree this would be awesome, the runtime cost of doing this is not something we can bear. We'll leave it as an exercise to the community :)
